target/generated-sources/annotations/mate/academy/bookstore/mapper/impl/BookMapperImpl.java

```diff
@@ -0,0 +1,127 @@
+package mate.academy.bookstore.mapper.impl;
+
+import java.math.BigDecimal;
+import javax.annotation.processing.Generated;
+import mate.academy.bookstore.dto.book.BookDto;
+import mate.academy.bookstore.dto.book.BookWithoutCategoryDto;
+import mate.academy.bookstore.dto.book.CreateBookRequestDto;
+import mate.academy.bookstore.mapper.BookMapper;
+import mate.academy.bookstore.model.Book;
+import org.springframework.stereotype.Component;
+
+@Generated(
+    value = "org.mapstruct.ap.MappingProcessor",
+    date = "2024-05-06T13:16:00+0300",
+    comments = "version: 1.5.5.Final, compiler: javac, environment: Java 19.0.2 (Oracle Corporation)"
+)
+@Component
+public class BookMapperImpl implements BookMapper {
```

Comment:
Please, add these to gitignore 
target/
!**/src/main/**/target/
!**/src/test/**/target/